ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-3node-ranGen.yaml

```diff
@@ -34,6 +34,11 @@ spec:
       metadata:
         name: openshift-node-performance-profile
       spec:
+        additionalKernelArgs:
+          - "idle=poll"
+          - "rcupdate.rcu_normal_after_boot=0"
+          - "kernel.timer_migration=1"
```

Comment:
Does this correctly set the sysctl value when the node is booted?

Reply:
No actually! Brought back Tuned with this value under [sysctl]